GoogleForCreators/web-stories-wp
# Video Optimization: MOV placeholder element position wrong

## Context



## Summary

Remove placement odd placement of placeholder videos on page. 



## Relevant Technical Choices

Remove all special logic to place placeholders. Let insertElement do it's thing and workout the best place to put the element. This bring drag into canva inline with insert from media library and insert from media picker. 

## To-do



## User-facing changes

#### Before 

<img width="1884" alt="Screenshot 2021-06-09 at 11 16 26" src="https://user-images.githubusercontent.com/237508/121338281-44c21b80-c915-11eb-87d6-8cbc59a1c7e3.png">

#### After

<img width="1884" alt="Screenshot 2021-06-09 at 11 14 24" src="https://user-images.githubusercontent.com/237508/121338244-3b38b380-c915-11eb-8469-01b4cbac3843.png">

## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Open editor. 
2. Drag .mov into canvas
3. See placement of video into canvas.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #7839
